app/templates/courses/teacher-classes-view.jade

```diff
@@ -127,3 +131,70 @@ mixin archivedClassRow(classroom)
     .col-xs-2
       .class-links.pull-right
         a.unarchive-classroom.text-h6(data-i18n='teacher.unarchive_class' data-classroom-id=classroom.id)
+
+mixin teacher-quests
+  - var numComplete = 0;
+  - var totalQuests = 0;
```

Comment:
Faster doesn't matter when the object is only 10 elements long though. `_.size` also does not enumerate the object.

For me at least, functional definitions of variables are much more clear than assembling the correct value across several lines that also do other things. The latter is also easier to mess up with later code changes.